tests/cxx/var_cleanup_attr.cc

```diff
@@ -0,0 +1,33 @@
+//===--- var_cleanup_attr.c - test input file for iwyu -------------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
```

Comment:
As mentioned in discussion, I agree that these tests are better suited for `tests/c`. Sorry about the back and forth.